### DISCUSSION TITLE
Strange JS Errors - Website Not Working

### DISCUSSION THREAD

**Discussion by simonferndriger** [alpinejs/alpine#3799] [1. Help]
I hope this is a rare case, at least I only witnessed it once so far (however, it was reproducable). The effect is, that the whole website stays blank basically, since not even the basic (Alpine) JS can get executed. I got the following JS errors tracked (same IP), but since they all occured at the same time, I'm not sure which one is exactly the cause of this issue:

![image](https://github.com/alpinejs/alpine/assets/1435963/f2f56bb6-4e46-4fbb-bdf4-c3db965dc3ab)

Currently I'm leaning towards the security error, which I made public: https://ferndriger-business.sentry.io/share/issue/cdfbcf2e9db044a28f39aa32d9288ae2/ -> Seems to have to do something with the `AlpineI18n` plugin?

The device is  up-to-date, so an outdated system is not the issue:

![image](https://github.com/alpinejs/alpine/assets/1435963/c0e92c4e-be7c-4001-9ed7-0b86edeb0f7a)

Any idea on how to approach this?

**Comment by ekwoka** [comment]
Are you using the Persist plugin? I believe this is the same error that is thrown if the user has disabled localstorage.

**Reply by simonferndriger** [reply]
Yes, I'm using that plugin too. Oooh, OK. So can't this just be handled by not using locastorage and let the app work nevertheless?

**Reply by ekwoka** [reply]
Currently, Persist still attempts to access localStorage regardless.

I believe there is a PR in progress that attempts to solve this: https://github.com/alpinejs/alpine/pull/3775

You can patch your local if you want to.

**Reply by simonferndriger** [reply]
Great - since it's a rare case, I can wait for the official fix - if it will be released somewhen this year or starting next year?

**Reply by ekwoka** [reply]
I would assume sometime soon 🤷 

I'm not a maintainer.
